
<html>
  <head>
    <title>Lab 10 - Graded Solution</title>
  </head>
  <body>
    <h1>Lab 10 - Graded Solution</h1>
    <p><b>You scored 0% = 0.0 points for this lab.</b></p>
    <ul>
      <li><b>test_ll_fix_simple</b>
      <ul>
      <li>Grade Percentage: 10</li>
      <li>Your solution failed this test case! taking 0.090 sec</Li>
<li><pre>IndentationError: expected an indented block (<tested-program>, line 20)</pre></li>
<li><pre>Traceback (most recent call last):
  File "/Users/lord_pretzel/.pyenv/versions/3.9.0/lib/python3.9/site-packages/grader-0.0.1-py3.9.egg/grader/execution_base.py", line 60, in call_test_function
    test_function(
  File "/Users/lord_pretzel/.pyenv/versions/3.9.0/lib/python3.9/site-packages/grader-0.0.1-py3.9.egg/grader/core.py", line 47, in wrapper
    raise module.caughtException
  File "/Users/lord_pretzel/.pyenv/versions/3.9.0/lib/python3.9/site-packages/grader-0.0.1-py3.9.egg/grader/program_container.py", line 38, in run
    self._exec_code()
  File "/Users/lord_pretzel/.pyenv/versions/3.9.0/lib/python3.9/site-packages/grader-0.0.1-py3.9.egg/grader/program_container.py", line 53, in _exec_code
    code = compile(source, "<tested-program>", "exec", dont_inherit=True)
  File "<tested-program>", line 20
    @staticmethod
    ^
IndentationError: expected an indented block
</pre></li>
      </ul>
      </li>
      <li><b>test_rr_fix_simple</b>
      <ul>
      <li>Grade Percentage: 10</li>
      <li>Your solution failed this test case! taking 0.093 sec</Li>
<li><pre>IndentationError: expected an indented block (<tested-program>, line 20)</pre></li>
<li><pre>Traceback (most recent call last):
  File "/Users/lord_pretzel/.pyenv/versions/3.9.0/lib/python3.9/site-packages/grader-0.0.1-py3.9.egg/grader/execution_base.py", line 60, in call_test_function
    test_function(
  File "/Users/lord_pretzel/.pyenv/versions/3.9.0/lib/python3.9/site-packages/grader-0.0.1-py3.9.egg/grader/core.py", line 47, in wrapper
    raise module.caughtException
  File "/Users/lord_pretzel/.pyenv/versions/3.9.0/lib/python3.9/site-packages/grader-0.0.1-py3.9.egg/grader/program_container.py", line 38, in run
    self._exec_code()
  File "/Users/lord_pretzel/.pyenv/versions/3.9.0/lib/python3.9/site-packages/grader-0.0.1-py3.9.egg/grader/program_container.py", line 53, in _exec_code
    code = compile(source, "<tested-program>", "exec", dont_inherit=True)
  File "<tested-program>", line 20
    @staticmethod
    ^
IndentationError: expected an indented block
</pre></li>
      </ul>
      </li>
      <li><b>test_lr_fix_simple</b>
      <ul>
      <li>Grade Percentage: 10</li>
      <li>Your solution failed this test case! taking 0.089 sec</Li>
<li><pre>IndentationError: expected an indented block (<tested-program>, line 20)</pre></li>
<li><pre>Traceback (most recent call last):
  File "/Users/lord_pretzel/.pyenv/versions/3.9.0/lib/python3.9/site-packages/grader-0.0.1-py3.9.egg/grader/execution_base.py", line 60, in call_test_function
    test_function(
  File "/Users/lord_pretzel/.pyenv/versions/3.9.0/lib/python3.9/site-packages/grader-0.0.1-py3.9.egg/grader/core.py", line 47, in wrapper
    raise module.caughtException
  File "/Users/lord_pretzel/.pyenv/versions/3.9.0/lib/python3.9/site-packages/grader-0.0.1-py3.9.egg/grader/program_container.py", line 38, in run
    self._exec_code()
  File "/Users/lord_pretzel/.pyenv/versions/3.9.0/lib/python3.9/site-packages/grader-0.0.1-py3.9.egg/grader/program_container.py", line 53, in _exec_code
    code = compile(source, "<tested-program>", "exec", dont_inherit=True)
  File "<tested-program>", line 20
    @staticmethod
    ^
IndentationError: expected an indented block
</pre></li>
      </ul>
      </li>
      <li><b>test_rl_fix_simple</b>
      <ul>
      <li>Grade Percentage: 10</li>
      <li>Your solution failed this test case! taking 0.073 sec</Li>
<li><pre>IndentationError: expected an indented block (<tested-program>, line 20)</pre></li>
<li><pre>Traceback (most recent call last):
  File "/Users/lord_pretzel/.pyenv/versions/3.9.0/lib/python3.9/site-packages/grader-0.0.1-py3.9.egg/grader/execution_base.py", line 60, in call_test_function
    test_function(
  File "/Users/lord_pretzel/.pyenv/versions/3.9.0/lib/python3.9/site-packages/grader-0.0.1-py3.9.egg/grader/core.py", line 47, in wrapper
    raise module.caughtException
  File "/Users/lord_pretzel/.pyenv/versions/3.9.0/lib/python3.9/site-packages/grader-0.0.1-py3.9.egg/grader/program_container.py", line 38, in run
    self._exec_code()
  File "/Users/lord_pretzel/.pyenv/versions/3.9.0/lib/python3.9/site-packages/grader-0.0.1-py3.9.egg/grader/program_container.py", line 53, in _exec_code
    code = compile(source, "<tested-program>", "exec", dont_inherit=True)
  File "<tested-program>", line 20
    @staticmethod
    ^
IndentationError: expected an indented block
</pre></li>
      </ul>
      </li>
      <li><b>test_key_order_after_ops</b>
      <ul>
      <li>Grade Percentage: 30</li>
      <li>Your solution failed this test case! taking 0.071 sec</Li>
<li><pre>IndentationError: expected an indented block (<tested-program>, line 20)</pre></li>
<li><pre>Traceback (most recent call last):
  File "/Users/lord_pretzel/.pyenv/versions/3.9.0/lib/python3.9/site-packages/grader-0.0.1-py3.9.egg/grader/execution_base.py", line 60, in call_test_function
    test_function(
  File "/Users/lord_pretzel/.pyenv/versions/3.9.0/lib/python3.9/site-packages/grader-0.0.1-py3.9.egg/grader/core.py", line 47, in wrapper
    raise module.caughtException
  File "/Users/lord_pretzel/.pyenv/versions/3.9.0/lib/python3.9/site-packages/grader-0.0.1-py3.9.egg/grader/program_container.py", line 38, in run
    self._exec_code()
  File "/Users/lord_pretzel/.pyenv/versions/3.9.0/lib/python3.9/site-packages/grader-0.0.1-py3.9.egg/grader/program_container.py", line 53, in _exec_code
    code = compile(source, "<tested-program>", "exec", dont_inherit=True)
  File "<tested-program>", line 20
    @staticmethod
    ^
IndentationError: expected an indented block
</pre></li>
      </ul>
      </li>
      <li><b>test_stress_testing</b>
      <ul>
      <li>Grade Percentage: 30</li>
      <li>Your solution failed this test case! taking 0.076 sec</Li>
<li><pre>IndentationError: expected an indented block (<tested-program>, line 20)</pre></li>
<li><pre>Traceback (most recent call last):
  File "/Users/lord_pretzel/.pyenv/versions/3.9.0/lib/python3.9/site-packages/grader-0.0.1-py3.9.egg/grader/execution_base.py", line 60, in call_test_function
    test_function(
  File "/Users/lord_pretzel/.pyenv/versions/3.9.0/lib/python3.9/site-packages/grader-0.0.1-py3.9.egg/grader/core.py", line 47, in wrapper
    raise module.caughtException
  File "/Users/lord_pretzel/.pyenv/versions/3.9.0/lib/python3.9/site-packages/grader-0.0.1-py3.9.egg/grader/program_container.py", line 38, in run
    self._exec_code()
  File "/Users/lord_pretzel/.pyenv/versions/3.9.0/lib/python3.9/site-packages/grader-0.0.1-py3.9.egg/grader/program_container.py", line 53, in _exec_code
    code = compile(source, "<tested-program>", "exec", dont_inherit=True)
  File "<tested-program>", line 20
    @staticmethod
    ^
IndentationError: expected an indented block
</pre></li>
      </ul>
      </li>
    </ul>
  </body>
</html>
